src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,32 @@
+.catalog {
+  @include centering;
+  grid-template-columns: repeat(1, $cardWidth);
+  gap: 48px;
+
+  @media (min-width:488px) {
+    grid-template-columns: repeat(2,$cardWidth);
+  }
+
+  @media (min-width:768px) {
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+
+  @media (min-width:1024px) {
+    grid-template-columns: repeat(4, $cardWidth);
+  }
+
+  &__card {
+    @include centering;
+    width: 200px;
+    height: 300px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    margin-bottom: 48px;
+    transition: color $transitionduration, transform $transitionduration ;
+  }
```

Comment:
also you can use "transition: all 0.3s" for all properties

Reply:
@Alireza-Mohammadhossein actually it is considered to be a not really good practice to use `transition: all`, better to write properties explicitly

It could be done as better though:

```
transition-property: color, transform;
transition-duration: $transitionDuration;
```